docker-compose.env.sample

```diff
@@ -9,6 +9,7 @@ AUTHORISATION_SERVICE_URL=
 AUTHORISATION_SERVICE_TOKEN=
 GOOGLE_DRIVE_JSON_KEY=
 GOOGLE_GEOCODING_API_KEY=
```

Comment:
If we're not using this anymore, can we drop it?

Reply:
I'd rather keep it around for now, as we might end up using it in #844

Reply:
Ok, fair enough.